flocker_bb/builders/flocker_vagrant.py

```diff
@@ -462,12 +462,39 @@ def slave_class(self):
 
 TUTORIAL_DISTRIBUTION = "centos-7"
 
+# FLOC-2437 @tomprince already removed centos-7 in
+# 07de6225a1cc35decaba66062e91cea42c791e11
+#
+# Ubuntu-14.04 should stay because it's a supported client platform.
+# It's documented in:
+#  * http://doc-dev.clusterhq.com/using/installing/index.html#installing-flocker-cli
+#
+# Apparently the only reason for running these tests on both rackspace and
+# aws, is to increase the chances of getting a green client installation build
+# when one of the cloud providers is running into api / resource limits.
+#
+# So what I can do is drop the rackspace provider below, which should reduce
+# the number of requests and the resource usage on our Rackspace cloud.
```

Comment:
I think another reason for running on both Rackspace and AWS is that they differ slightly in what they offer.  Both are "Ubuntu 14.04" (or Ubuntu 15.04) I suppose ... just not quite the same.  Generally speaking, testing multiple configurations is a good thing.  Not necessarily because we want the CLI to work on a Rackspace Ubuntu 14.04 instance (though it's nice if it does) but because it will show us if we come to implicitly depend on a feature of one configuration that's not generally available (so testing on AWS and Rackspace probably makes it slightly more likely that the CLI will work on someone's actual, real, normal desktop installation ... though not as much more likely as if we tested that configuration :smile: ).

That said ...